spec/index.bs

```diff
@@ -3583,6 +3580,9 @@ interface GPUComputePassEncoder {
     void dispatch(GPUSize32 x, optional GPUSize32 y = 1, optional GPUSize32 z = 1);
     void dispatchIndirect(GPUBuffer indirectBuffer, GPUSize64 indirectOffset);
 
+    void beginPipelineStatisticsQuery(GPUQuerySet querySet, GPUSize32 queryIndex);
+    void endPipelineStatisticsQuery(GPUQuerySet querySet, GPUSize32 queryIndex);
```

Comment:
Actually, since pipeline statistics queries can't be nested, could this just be `endPipelineStatisticsQuery()`?

Reply:
Yes, I will remove these args.